packages/matchbox/src/components/RadioCard/styles.js

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components';
+import { tokens } from '@sparkpost/design-tokens';
+import { focusOutline, visuallyHidden } from '../../styles/helpers';
+import { Box } from '../Box';
+
+export const StyledLabel = styled(Box)`
+  display: block;
+  position: relative;
+  padding: ${props => props.theme.space['500']};
+  border: ${props => props.theme.borders['400']};
+  border-radius: ${props => props.theme.radii['200']};
+  cursor: pointer;
+  transition: ${tokens.motionDuration_medium};
```

Comment:
Should we add this to the theme?

Reply:
I'm not sure, styled-system doesn't have a category for transitions, and they don't expose them as system props so I decided not to